packages/keyboard-shortcuts/src/store/actions.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Returns an action object used to register a new keyboard shortcut.
+ *
+ * @param {string} name        Shortcut name.
+ * @param {string} category    Shortcut category.
+ * @param {string} description Shortcut description.
+ * @param {Object} combination Shortcut keys combination.
+ * @param {Array}  aliases     Shortcut aliases.
+ */
+export function registerShortcut( name, category, description, combination, aliases ) {
```

Comment:
Observation: I was looking at specific usages of `registerShortcut` before seeing this function signature, and it was very unclear to me what the specific arguments were intended to represent, as they were positional. While it's more verbose to do so, an object of properties could help clarify what they're intended to represent, and could allow for more options to be added in the future without too much trouble.

Reply:
I was expecting this comment and yes, you're right, it started with just 2 arguments, now it's obvious that we need an object.